test.py

```diff
@@ -14,7 +14,7 @@
 import traceback
 import unittest2 as unittest
 
-flags = {'form': 1, 'pynwb': 2, 'integration': 3, 'example': 4}
+flags = {'pynwb': 2, 'integration': 3, 'example': 4}
```

Comment:
Very minor, nitpicky thing. Should this change to ``flags = {'pynwb': 1, 'integration': 2, 'example': 3}`` just to have the numbering of options start at 1. Just ignore if this causes trouble.

Reply:
I missed this after already merged... I'll get it on the next fix

Reply:
No problem. Its such a minor thing anyways.